app/src/test/java/org/simple/clinic/user/clearpatientdata/SyncAndClearPatientDataTest.kt

```diff
@@ -34,20 +34,6 @@ class SyncAndClearPatientDataTest {
   private val prescriptionPullToken = mock<Preference<Optional<String>>>()
   private val medicalHistoryPullToken = mock<Preference<Optional<String>>>()
 
-  private val syncAndClearPatientData = SyncAndClearPatientData(
```

Comment:
Curious, why are we doing this?

Reply:
In a later commit, I need to override one of the properties passed to the constructor. So I made this change.